requirements-dev.txt

```diff
@@ -2,6 +2,8 @@ wheel==0.37.0
 behave==1.2.5
 jsonschema==2.5.1
 coverage==5.5
+importlib_resources==5.8.0
```

Comment:
I don't think we want to add this to the dev dependencies. Was there something specific we needed?

Reply:
yeah im removing now